src/test/java/uk/gov/pay/api/resources/GetOnePaymentStrategyTest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.pay.api.resources;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.LoggingEvent;
+import ch.qos.logback.core.Appender;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.TokenPaymentType;
+import uk.gov.pay.api.service.GetPaymentService;
+
+import java.util.List;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetOnePaymentStrategyTest {
+    
+    @Mock
+    private GetPaymentService mockGetPaymentService;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
It's never used

Reply:
good spot - was supposed to remove it, but forgot.